vspace/src/main/java/edu/asu/diging/vspace/core/references/ReferenceBlock.java

```diff
@@ -0,0 +1,33 @@
+package edu.asu.diging.vspace.core.references;
```

Comment:
It looks like this class is only be used by the controller. It should therefore be in the web package somewhere.